packages/react/src/patterns/sections/LeadSpace/__stories__/LeadSpace.stories.js

```diff
@@ -68,15 +69,15 @@ storiesOf('Patterns (Sections)|LeadSpace', module)
     };
 
     return (
-      <div className="bx--grid">
+      <DotcomShell footerType="short">
```

Comment:
@RaphaelAmadeu I think in huddle we decided to keep the LeadSpace isolated in storybook.